packages/sdk/src/server/index.ts

```
@@ -353,7 +359,8 @@ export class BackendClient {
    * @param opts - The options for configuring the server client.
    */
   constructor(opts: BackendClientOpts) {
-    this.environment = opts.environment ?? "production";
+    this.ensureValidEnvironment(opts.environment)
+    this.environment = opts.environment!!
```

Biome

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)